src/components/OCNotificationMessage.vue

```diff
@@ -68,17 +64,46 @@ export default {
       required: false,
       default: null,
     },
+    /**
+     * Number of seconds the message shows. It will disappear after this time. Setting the timeout to 0 while the
+     * notification is dismissible will bypass the timeout.
+     */
+    timeout: {
+      type: Number,
+      required: false,
+      default: 5,
+    },
+    /**
+     * Property weather a close button will be shown or not
```

Comment:
```suggestion
     * Whether a close button will be shown or not
```

Reply:
"Solved" due to the removal of this property.